src/App.vue

```diff
@@ -314,7 +352,7 @@ export default {
   },
 
   /** Called just before this component is destroyed. */
-  beforeDestroy (): void {
+  beforeUnmount (): void {
```

Comment:
Please test this.

You can add a `debugger` command in here to force a breakpoint, or you could add an alert (which stops execution).

(In Edit UI, I left it as beforeDestroy(). Hopefully that's correct.)

Reply:
So I tested this with both `beforeUnmount` and `beforeDestroy` by disabling the `vue/no-deprecated-destroyed-lifecycle` rule. Unless I'm missing something, they seem to work the exact same way. Shall I keep it `beforeUnmount` or change it to `beforeDestroy` with the rule so that we are consistent with Edit UI?

Reply:
Cool. Thanks for looking into it.

Keep it as `beforeUnmount` since that appears to be the future way. Ref: https://vuejs.org/guide/essentials/lifecycle.html#lifecycle-diagram

We can deal with Edit UI later. Interesting that it doesn't complain about `beforeDestroy` even though it doesn't have the rule override...?

Reply:
Confused? -> I don't see this same lint issue in Edit UI, even though it uses the old lifecycle event but doesn't have the rule to shut up the message. I'm confused, too, and suggesting we don't worry about it for now.

Reply:
Exactly! I've trying to figure out why this might be the case. Could it be since Filings UI uses Options API while Edit UI uses Class API? I really don't know.

Reply:
You're probably right 😃